alphadia/libtransform.py

```diff
@@ -314,6 +314,11 @@ def forward(self, input: SpecLibBase) -> SpecLibBase:
 
         model_mgr = ModelManager(device=device)
         if self.checkpoint_folder_path is not None:
+            if not os.path.exists(self.checkpoint_folder_path):
```

Comment:
is `checkpoint` really a good name? I confused it with the `.progress` concept ..

Reply:
This is how it's called within peptdeep at the moment. We could name it `peptdeep_model_path`, what do you think?

Reply:
sounds good